index.html

```diff
@@ -5838,6 +5838,20 @@ <h3>Group Control</h3>
         </p>
       </section>
     </section>
+    <section>
+      <h2>Changing the DID subject</h2>
+      <p>
+A <a>DID document</a> has exactly one <a>DID subject</a>, identified by exactly
+one <a>DID</a>, expressed as the value of the the <code><a>id</a></code>
+property. This property value is immutable for the lifetime of the
```

Comment:
I think the trouble can be resolved like this:

```suggestion
A <a>DID document</a> has exactly one <a>DID</a>, which refers to
a <a>DID subject</a>. The <a>DID</a> is expressed as the value of the
<code><a>id</a></code> property. This property value is immutable for
the lifetime of the
```

Reply:
Yeah I'd be good with this change as well.